Realm/RLMSyncUtil.h

```diff
@@ -59,6 +65,16 @@ typedef RLM_ERROR_ENUM(NSInteger, RLMSyncError, RLMSyncErrorDomain) {
 
     /// An error that indicates an internal, unrecoverable error with the underlying synchronization engine.
     RLMSyncErrorClientInternalError     = 6,
+
+    /// An error that indicates the Realm needs to be reset.
```

Comment:
Doc comments should have an empty line between the summary and extended description (but still with the /// prefix).